framework/include/functions/MooseParsedFunctionWrapper.h

```diff
@@ -104,6 +105,12 @@ class MooseParsedFunctionWrapper
   /// Vector of pointers to PP values
   std::vector<Real *> _scalar_vals;
 
+  /// Stores the relative location of variables (in _vars) that are connected to Functions
+  std::vector<unsigned int> _fn_index;
+
+  /// Vector of functions
+  std::vector<Function *> _fns;
```

Comment:
Not my favorite class scoped variable name 😦

Reply:
But, it nicely matches `_pp_vals` and `_pp_index` a few lines above 🙃